src/test/kotlin/MainTest.kt

```diff
@@ -0,0 +1,40 @@
+import kotlinx.cli.*
+import kotlinx.cli.ArgParser
+import kotlinx.cli.ArgType
+import kotlinx.cli.required
+import kotlinx.coroutines.runBlocking
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.Test
+
+class KotlinCliTest {
```

Comment:
These tests don't make any sense: they aren't using the code in `Main.kt` or `Mempool.kt`, you instead copy-pasted it here which isn't useful at all. The issue is that you have currently put your business logic directly inside the `main` function in `Main.kt`, which isn't a good idea. We will discuss it during today's call.